field_float_decimal_place/models/field_float_decmial.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class FieldFloatDecimal(models.Model):
+    _name = "field.float.decimal"
+    _description = "Field Float Decimal"
+    _order = "res_model_id, field_id"
+
+    res_model_id = fields.Many2one(
+        "ir.model", string="Model", ondelete="cascade", required=True
+    )
+    res_model_name = fields.Char("Model Name", related="res_model_id.model", store=True)
+    field_id = fields.Many2one(
```

Comment:
Shouldn't we make it `res_field_id`, or keep this as is and change `res_model_id` to `model_id` to be consistent?